index.js

```diff
@@ -81,6 +81,20 @@ function ternaryDispatcher(wrap, f) {
   };
 }
 
+function fantasyLandUnaryDispatcher(wrap, f) {
```

Comment:
You don't really need in inject the wrapper, as every Fantasy Land method always returns a Future, you can just in-line the `fluent` call.